pkg/formatters/rawFormatter.go

```diff
@@ -41,7 +41,7 @@ func (*RawFormatter) FormatRuns(runs []galasaapi.Run, apiServerUrl string) (stri
 
 		duration := getDuration(startTimeStringRaw, endTimeStringRaw)
 
-		runLog := apiServerUrl + "/ras/run/" + run.GetRunId() + "/runlog"
+		runLog := apiServerUrl + RAS_RUNS_URL + run.GetRunId() + "/runlog"
```

Comment:
This information should be coming from a REST response payload field from the servlet, rather than being made-up/inferred by the CLI what it should be...

Rest users could do with this URL information also.

Reply:
And when we have a UI which can display runlogs better, we will switch it to point to that. Would be nice not to have to change the CLI at that point.